### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by stvowi** [atatanasov/gijgo#83]
It would be great great to have a documentation on the Gijgo Homepage of the component properties wich ccan configured via HTML data-* attributes.

**Comment by atatanasov** [comment]
You can configure every option with data-* attribute. We don't have any limitations. Do you have some issues with any particular data-* attribute?

**Comment by stvowi** [comment]
I've seen in the js source code, that the data-*attributes are documented. The property names and the data-* names are different. Is there a naming convention ? (Camelcase name (e.g. abcDef ) to name with dash ( eg. abc-def)

**Comment by atatanasov** [comment]
Yes. You assumpsion is correct. The convension in camel case name to the lower case name split with dashes. I will extend the configuration in the near future in order to avoid confusions. I wil leave this ticket open for now. I will close it after the extension of the documentation.
